flags/flags.go

```diff
@@ -0,0 +1,17 @@
+package flags
+
+import (
+	"github.com/cosmos/cosmos-sdk/client/flags"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/pkg/consts"
+)
+
+// List of CLI flags
+const FlagSquareSizes = "square-sizes"
+
+// AddTxFlagsToCmd adds common flags to a module tx command.
+func AddTxFlagsToCmd(cmd *cobra.Command) {
+	flags.AddTxFlagsToCmd(cmd)
+	cmd.Flags().UintSlice(FlagSquareSizes, []uint{consts.MaxSquareSize, 128, 64}, "Specify the square sizes, must be power of 2")
+}
```

Comment:
This instance isn't particularly bad, but wrapping existing commands like this can be misleading to future readers, and is typically not a good practice. If readers are familar with the sdk, then they will assume that this only does what the sdk's function does, but that's not the case here. We're implicitly adding a flag. This function doesn't just add the Tx subcommand, it also adds the square-sizes flag. 

Instead, I think we should aim to be explicit. There's no reason to hide that we're adding the square sizes command, in fact, we literally want to tell the reader that that's what we're doing. The simplest way to do this is right below [this line](https://github.com/celestiaorg/celestia-app/blob/63519eccc053e9695af5901164dfd5da5fe3fb24/x/payment/client/cli/wirepayformessage.go#L104) using the standard way to add flags (example in [dalc](https://github.com/celestiaorg/dalc/blob/d62d3a2a19c2d08657cf0e55788c38f4e9accda9/cmd/dalc/main.go#L81) (except we don't want to use strings))

this has the added benefit of not have an entire extra package dedicated to a few flags, and instead keeps the flags close to where they will be used, which is usually a good thing.

Reply:
Great, I moved this into `cli/wirepayformessage.go` so we don't need the `flags.go` anymore.